app/operations/placements/change_status.rb

```diff
@@ -40,4 +50,12 @@ def save_placement(placement)
   rescue ActiveRecord::RecordInvalid => e
     Failure[:placement_invalid, placement.errors.full_messages.presence || e.to_s]
   end
+
+  def find_disqualify_reason(reason)
+    disqualify_reason = DisqualifyReason.find_by(title: reason.humanize)
```

Comment:
Same question about humanize. Actually, better to send "id" to backend if it's possible

Reply:
The problem with id is that status can be `qualified`/`reserved` and then it won't have a disqualify_reason. The only workaround I see is to pass 2 arguments: 'status' and "id' (that can be nil) instead of one